src/utils/datetime.js

```diff
@@ -3,7 +3,9 @@
  * @file Date and time utilities
  * @author Box
  */
+import * as React from 'react';
```

Comment:
I don't think we should introduce React components into our utils. If this is necessary, the result is really not a util anymore and should be moved to its own component.

Reply:
Can this use the `ReadableTime` component we are using for comments, tasks, etc?